pyproject.toml

```diff
@@ -3,13 +3,17 @@ name = "open-payments-sdk"
 version = "0.1.0"
 description = "Python SDK implementation for open-payments API"
 authors = [
-    {name = "Yiannis Giannelos",email = "johngiannelos@gmail.com"}
+    {name = "Yiannis Giannelos",email = "johngiannelos@gmail.com"},
+    {name = "Elijah Okello", email = "elijahokello90@gmail.com"},
+    {name = "Kasweka Michael Mukoko", email = "kasweka.mukoko@izyane.com"}
 ]
 license = {text = "Apache-2.0"}
 requires-python = ">=3.11"
 dependencies = [
     "httpx (>=0.28.1,<0.29.0)",
```

Comment:
There doesn't seem to be any vulnerabilities with these dependencies, but could we also run the `pip-audit` as part of the workflow?

```yaml
jobs:
  audit:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - name: Install dependencies
        run: |
          python3 -m pip install .  # assumes dependencies declared in pyproject.toml
          python3 -m pip install pip-audit
      - name: Run pip-audit
        run: pip-audit .
```